src/extensions/ttt.py

```diff
@@ -0,0 +1,136 @@
+from discord.ext import commands
+import discord
+
+import random
+
+
+players = [":x:", ":o:"]
+wordboard = [":one:", ":two:", ":three:",
+             ":four:", ":five:", ":six:",
+             ":seven:", ":eight:", ":nine:"]
+
+
+class tictactoe(commands.Cog):
+    def __init__(self, bot):
+        self.client = bot
+        self.__reset_board()
+        self.gameover = True
+
+    def __print_board(self):
+        line = ""
+        for i in range(3):
+            for j in range(3):
+                if self.__is_number(self.board[i*3 + j]):
+                    line += wordboard[i*3 + j]
+                else:
+                    line += self.board[i*3 + j]
+            line += "\n"
+        return line
+
+    def __reset_board(self):
+        self.board = [str(i+1) for i in range(9)]
+        self.currentplayer = random.choice(players)
+        self.gameover = False
+
+    def __check_win(self):
+        board = self.board
+        # diagonal:
+        if ((board[0] == board[4] == board[8]) or (board[2] == board[4] == board[6])) and not self.__is_number(board[4]):
+            return True
+        # check columns:
+        for i in range(3):
+            if board[i] == board[i+3] == board[i+6] and not self.__is_number(board[i]):
+                return True
+        # check rows:
+        for i in range(3):
+            if board[i*3] == board[i*3 + 1] == board[i*3 + 2] and not self.__is_number(board[i*3]):
+                return True
+        return False
+
+    def __check_tie(self):
+        for tile in self.board:
+            if self.__is_number(tile):
+                return False
+        return True
+
+    def __is_number(self, value):
```

Comment:
First of all, this should be using a more specific Exception. Second of all, this seems like a completely unnecessary function.
Python has ```str.isnumeric()``` which should suffice.

If you really must have this function, I'd prefer a ternary operator.

Reply:
Removed the method, replaced it with python's alternative in the latest commit